pkg/chunk/series_store.go

```diff
@@ -272,23 +272,20 @@ func (c *seriesStore) lookupSeriesByMetricNameMatchers(ctx context.Context, from
 	if shard != nil {
 		matchers = append(matchers[:shardLabelIndex], matchers[shardLabelIndex+1:]...)
 	}
-
-	// Just get series for metric if there are no matchers
-	if len(matchers) == 0 {
-		indexLookupsPerQuery.Observe(1)
-		series, err := c.lookupSeriesByMetricNameMatcher(ctx, from, through, userID, metricName, nil, shard)
-		if err != nil {
-			preIntersectionPerQuery.Observe(float64(len(series)))
-			postIntersectionPerQuery.Observe(float64(len(series)))
-		}
-		return series, err
-	}
+	counter := 0
```

Comment:
"counter" is a bit generic; better to call this something like `matchersCount` or `nMatchers`

Reply:
Done!